Maquetado/scss/footer.scss

```diff
@@ -0,0 +1,24 @@
+.footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column nowrap;
+  padding: 50px 15px 15px 10px;
```

Comment:
You  15px in the right and 10px in the left. You need to used the same value for things to be correctly centered